cli.go

```diff
@@ -290,6 +295,41 @@ func callRPCListTopicPeer(rpcAddr string, topics []string) {
 	fmt.Println(topicPeersString)
 }
 
+func callRPCListShardPeer(rpcAddr string, shards []string) {
+	conn, err := grpc.Dial(rpcAddr, grpc.WithInsecure())
+	if err != nil {
+		logger.Fatalf("Failed to connect to RPC server at %v, err: %v", rpcAddr, err)
+	}
+	defer conn.Close()
+	client := pbrpc.NewPocClient(conn)
+
+	topics := make([]string, len(shards))
+	for i := 0; i < len(shards); i++ {
+		topics[i] = "shardCollations_" + shards[i]
+	}
+
+	listTopicPeerReq := &pbrpc.RPCListTopicPeerRequest{
+		Topics: topics,
+	}
+	logger.Debugf("rpcclient:ListTopicPeer: sending=%v", listTopicPeerReq)
+	res, err := client.ListTopicPeer(context.Background(), listTopicPeerReq)
+	if err != nil {
+		logger.Fatalf("Failed to call RPC ListTopicPeer at %v, err: %v", rpcAddr, err)
+	}
+	logger.Debugf("rpcclient:ListTopicPeer: result=%v", res)
+	shardPeers := make(map[string][]string)
+	for topic, peers := range res.TopicPeers {
+		var shard = strings.Replace(topic, "shardCollations_", "", -1)
```

Comment:
@mhchia same as line 308

Reply:
Maybe we can have a function transform shardTopics to `shardID`?

Reply:
10-4 will do